### PR TITLE
Feature - docker-compose polymer bundle [#53]

### LINKED ISSUE #53: Allow running the bundled version of the polymer app (+ the api, of course).

### PR DESCRIPTION
**Pull request by n1207n**
### This PR closes #53.

This PR contains a separate docker-compose yml file and updated package.json for building polymer bundle folder and serving lite-server from the bundle folder.

##### Feature list
* A new docker-compose yml file for creating polymer container as bundled version

* package.json update

* Disabled gulp-imagemin package

##### Progress checker
- [x] Polymer

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -29,6 +29,13 @@ def up():
     local('docker-compose up')
 
 
+def up_with_bundle():
```
Comment (chachra) [inline fabfile.py]: devup_built to keep consistent with eTools ? Or is that different thing?

**Comment by chachra** [comment]
To call this task complete @n1207n we need to have a way to get this to run in this mode on any cloud (test/demo/staging etc.) environment as well. Maybe not a code related thing, but can you confirm its that way on our cloud envs?

File: polymer/gulpfile.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ function source() {
     .pipe(gulpif('**/*.js', javascript.lint())).on('end', log('Linted Javascript'))
     .pipe(gulpif('**/*.js', javascript.minify())).on('end', log('Minified Javascript'))
 
-    .pipe(gulpif('**/*.{png,gif,jpg,svg}', images.minify())).on('end', log('Minified Images'))
+    // .pipe(gulpif('**/*.{png,gif,jpg,svg}', images.minify())).on('end', log('Minified Images'))
```
Comment (rkrupinski) [inline polymer/gulpfile.js]: Why is that?

File: polymer/package.json [hunk 3]
```diff
@@ -27,7 +29,7 @@
     "gulp-htmlhint": "^0.3.1",
     "gulp-htmlmin": "^2.0.0",
     "gulp-if": "^2.0.1",
-    "gulp-imagemin": "^3.0.1",
+    "gulp-imagemin": "^3.4.0",
```
Comment (rkrupinski) [inline polymer/package.json]: Why is that?

**Comment by chachra** [comment]
@rkrupinski caching issues are resolved on this PR? Can retest now?

**Comment by rkrupinski** [comment]
@chachra All should work now, although build time grew significantly.

**Comment by n1207n** [comment]
I ran `fab up_with_bundle` command and it came as error after a long time of waiting.

Killed
polymer_1     | npm ERR! code ELIFECYCLE
polymer_1     | npm ERR! errno 137
polymer_1     | npm ERR! polymer-starter-kit-custom-build@0.0.0 bundle: `node --max_old_space_size=2048 ./node_modules/gulp/bin/gulp`
polymer_1     | npm ERR! Exit status 137
polymer_1     | npm ERR! 
polymer_1     | npm ERR! Failed at the polymer-starter-kit-custom-build@0.0.0 bundle script 'node --max_old_space_size=2048 ./node_modules/gulp/bin/gulp'.

**Pull request merged**